### PR TITLE
LPS-159029 My Activities Page needs a refresh

### PR DESCRIPTION
**Pull request by TalitaFraga**

### CODE REVIEW

**Comment by TalitaFraga** [comment]
ci: test:sf

**Comment by TalitaFraga** [comment]
ci:test:sf

**Comment by TalitaFraga** [comment]
ci:test:sf

**Comment by TalitaFraga** [comment]
ci:test:sf

**Comment by Claudionilton** [comment]
ci:test:sf

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Comments.es.js [hunk 1]
```diff
@@ -82,6 +84,13 @@ export default withRouter(
 			]);
 
 			onSubscription({allowSubscription});
+
+			deleteCacheKey(getUserActivityQuery, {
+				filter: `creatorId eq ${context.userId}`,
+				page,
+				pageSize,
+				siteKey: context.siteKey,
+			});
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/Comments.es.js:93]: Hard coded values

```suggestion
			deleteCacheKey(getUserActivityQuery, {
				filter: `creatorId eq ${context.userId}`,
				page: 1,
				pageSize: 20,
				siteKey: context.siteKey,
			});
```

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js [hunk 2]
```diff
@@ -219,6 +220,13 @@ export default withRouter(
 				await onSubscription();
 
 				fetchMessages();
+
+				deleteCacheKey(getUserActivityQuery, {
+					filter: `creatorId eq ${context.userId}`,
+					page,
+					pageSize,
+					siteKey: context.siteKey,
+				});
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js:229]:
```suggestion
				deleteCacheKey(getUserActivityQuery, {
					filter: `creatorId eq ${context.userId}`,
					page: 1,
					pageSize: 20,
					siteKey: context.siteKey,
				});
```

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js [hunk 3]
```diff
@@ -549,6 +557,8 @@ export default withRouter(
 												editable={!question.locked}
 												key={answer.id}
 												onSubscription={onSubscription}
+												page={page}
+												pageSize={pageSize}
```
Comment (kevenleone) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/Question.es.js:561]:
```suggestion
											
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
@TalitaFraga , @Claudionilton could you please take a look? thanks

**Comment by TalitaFraga** [comment]
resent: https://github.com/pt-liferay-solutions/liferay-portal/pull/2070

**Pull request closed without merging**